core/include/vecmem/containers/data/jagged_vector_buffer.hpp

```diff
@@ -95,6 +98,16 @@ class jagged_vector_buffer : public jagged_vector_view<TYPE> {
                          memory_resource& resource,
                          memory_resource* host_access_resource = nullptr);
 
+    /// Copy constructor
+    jagged_vector_buffer(const jagged_vector_buffer&) = default;
```

Comment:
Crud... I was meaning to remove these copy constructors... :frowning: (These are not really well defined for the buffer types.) Shoot...